From 56333cc7dbfb80bf94c86b614f330a981be21258 Mon Sep 17 00:00:00 2001
From: Howard Hsu <howard-yh.hsu@mediatek.com>
Date: Thu, 20 Jun 2024 09:01:35 +0800
Subject: [PATCH 79/89] mtk: mac80211: add new rc changed enum for wifi7 cert

Add a new rc enum "IEEE80211_RC_CODING_TYPE_CHANGED" for wifi7 r1. This
enum is used to mark that the coding type has changed by user space
commands.

Signed-off-by: Howard Hsu <howard-yh.hsu@mediatek.com>

---
 include/net/mac80211.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/net/mac80211.h b/include/net/mac80211.h
index 9471892..f9afddd 100644
--- a/include/net/mac80211.h
+++ b/include/net/mac80211.h
@@ -3757,6 +3757,8 @@ enum ieee80211_rate_control_changed {
 	IEEE80211_RC_SMPS_CHANGED	= BIT(1),
 	IEEE80211_RC_SUPP_RATES_CHANGED	= BIT(2),
 	IEEE80211_RC_NSS_CHANGED	= BIT(3),
+	/* Defined for mtk vendor command */
+	IEEE80211_RC_CODING_TYPE_CHANGED= BIT(7),
 };
 
 /**
-- 
2.18.0

